index.html

```diff
@@ -630,71 +632,103 @@ <h1>Streaming GCN<br>Notices in Python</h1>
       </div>
     </section>
 
-<section data-markdown>
```

Comment:
This is a huge diff but it looks like mostly whitespace changes. Please revert the unrelated whitespace changes.

Reply:
Ok, I see what the cause is, but it looks like these slides were previously was weirdly not indented, and now it is indented to match the other sections.  Why is this an issue?

Reply:
A couple reasons. First of all, the formatting was very sensitive to indentation and there may have been a reason that this section was done differently. Second, it's important to keep merge requests focused and not have irrelevant material in them so that they are quick to review and can be reverted as individual functional units if they cause regressions.